tests/test_audio.py

```diff
@@ -156,6 +168,24 @@ def test_load_timestamp_before_warnmode(metadata_wav_str):
         assert s.metadata["recording_start_time"] == correct_ts
 
 
+def test_retain_metadata(metadata_wav_str, new_metadata_wav_str):
+    a = Audio.from_file(metadata_wav_str)
+    a.save(new_metadata_wav_str)
+    new_a = Audio.from_file(new_metadata_wav_str)
+
+    # file size may differ slightly, other fields should be the same
+    new_a.metadata["filesize"] = None
```

Comment:
Instead of testing one random field, could we not loop through all fields and verify.

Reply:
This tests all fields except filesize, because filesize can differ without any change in samples or metadata.